state-sync/state-sync-v1/src/executor_proxy.rs

```diff
@@ -246,88 +200,42 @@ impl ExecutorProxyTrait for ExecutorProxy {
             .map_err(|error| Error::UnexpectedError(error.to_string()))
     }
 
-    fn publish_on_chain_config_updates(&mut self, events: Vec<ContractEvent>) -> Result<(), Error> {
-        if events.is_empty() {
-            return Ok(());
-        }
+    async fn publish_events_and_reconfig_notifications(
+        &mut self,
```

Comment:
should this just take the version? we have it in the `verified_target_li` right?